src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -63,39 +83,66 @@ export class ReviewRoute extends Component<ReviewProps> {
     try {
       const orderOrError =
         this.props.order.mode === "BUY"
-          ? (await this.submitBuyOrder()).ecommerceSubmitOrder.orderOrError
-          : (await this.submitOffer()).ecommerceSubmitOrderWithOffer
-              .orderOrError
+          ? (await this.submitBuyOrder()).commerceSubmitOrder.orderOrError
+          : (await this.submitOffer()).commerceSubmitOrderWithOffer.orderOrError
 
       if (orderOrError.error) {
         this.handleSubmitError(orderOrError.error)
         return
+      } else if (
+        this.isOrder(orderOrError) &&
```

Comment:
this name is little confusing, are we trying to see if this order requires action? is it more like `this.orderRequiresAction`?

Reply:
This was to prevent type errors because offer responses will never have `actionData`. `isOrder` just checks to see if the order is an order and not an offer

Reply:
cool, makes sense, i would just rename it to `isBuyNowOrder` then, since offer's have Order too.